src/main/java/org/apache/sling/feature/maven/mojos/ApisJarMojo.java

```diff
@@ -1582,6 +1589,14 @@ private File createArchive(final ApisJarContext ctx,
             archiveConfiguration.addManifestEntry("Specification-Vendor", project.getOrganization().getName());
         }
 
+        // replace manifest entries with the one provided in properties configuration
+        Map<String,String> archiveConfigMap =  archiveConfiguration.getManifestEntries();
+        for (final String name: manifestProperties.stringPropertyNames()){
+            if(archiveConfigMap.containsKey(name))
```

Comment:
Why is this if needed? Without it, you could add properties to the manifest - and not just replace

Reply:
Actually this is for replacing only those values in generated Manifest file that are provided in _manifestProperties_ configuration, key/value not provided in configuration will be picked form default one. 

I added this only for purpose where user want to replace the original values generated by internal logic that we have , not to add new one but we can extend this to add new key/value pair too.

Reply:
ok, yes, I think it makes sense to allow adding additional key/value pairs